lib/models/finamp_models.dart

```diff
@@ -2193,8 +2193,10 @@ enum KeepScreenOnOption {
   @HiveField(0)
   disabled,
   @HiveField(1)
-  whilePlaying,
+  alwaysOn,
   @HiveField(2)
+  whilePlaying,
+  @HiveField(3)
```

Comment:
It's fine to do this now since the feature hasn't been released yet, but for existing features you shouldn't modify or re-assign these IDs, since that would unexpectedly change some settings for users.